### PR TITLE
LRQA-60285 Follow up

### PR DESCRIPTION
**Pull request by martamedio**
Follow up, see: https://github.com/brianchandotcom/liferay-portal/pull/97688#issuecomment-760894161

cc/ @gaborlovasdotcom @csierra

### CODE REVIEW

File: modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -7,5 +7,5 @@ multi-factor-authentication=Multi-Factor Authentication
 multi-factor-authentication-has-been-disabled-by-the-system-administrator-and-is-unavailable-to-all-instances=Multi-factor authentication has been disabled by the system administrator and is unavailable to all instances. You will be notified when it becomes enabled again so long as you keep it enabled in Instance Settings.
```
Comment (liferay-appsec) [inline modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-web/src/main/resources/content/Language.properties:7]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by martamedio** [comment]
ci:forward

**Comment by martamedio** [comment]
ci:forward

**Pull request closed without merging**